0.download-data/README.md

```diff
@@ -1,2 +1,79 @@
-# tcga-joint-representation
-Data Download Procedure
\ No newline at end of file
+# Introduction - Data Download Procedure
+
+The Genomic Data Commons (GDC) portal offers API access to download clinical, image and genomic data for patient cases.
+We will leverage Python to access the API endpoints and download case data related to TCGA.
+
+### Search Data
+
+We can query the GDC API using a wide set of parameters to return results of a search or details about a specific entity.
+The API provides helper features to assist the user in building their query and understand available fields.
+
+### Download Files
+
+The GDC API can be used to serve data to the user. 
+If restricted data is requested, a token must be provided by the user along with the API call. 
+This token can be downloaded directly from the GDC Data Portal or the GDC Data Submission Portal.
```

Comment:
These two sentences (lines 14 and 15) are still passive voice. Please convert to active voice :)